crd/client.go

```diff
@@ -1,3 +1,6 @@
+//go:build !ignore_crd
```

Comment:
i don't think we should ignore this package. the autogenerated stuff, yes, but i wrote these other files and they could (should) be tested

Reply:
@rbtr what about crd/nodenetworkconifg/client.go & embed.go?

Reply:
yes, those should also be included in the coverage stats because they should be tested